apps/gp2-server/src/routes/public/user.route.ts

```diff
@@ -39,24 +39,28 @@ export const userRouteFactory = (userController: UserController): Router => {
 const mapUserToPublicUser = (
   user: gp2Model.UserResponse,
 ): gp2Model.PublicUserResponse => ({
-  id: user.id,
-  displayName: user.displayName,
+  avatarUrl: user.avatarUrl,
+  biography: user.biography,
+  city: user.city,
   country: user.country,
   degrees: user.degrees,
+  displayName: user.displayName,
   firstName: user.firstName,
+  id: user.id,
   lastName: user.lastName,
-  publishDate: user.createdDate,
-  avatarUrl: user.avatarUrl,
-  biography: user.biography,
-  city: user.city,
+  lastModifiedDate: user.lastModifiedDate,
   middleName: user.middleName,
+  institution: user.positions[0]?.institution,
```

Comment:
Looking at the ticket, it seems we should display all institutions, right?
![Screenshot 2024-06-04 at 07 37 30](https://github.com/yldio/asap-hub/assets/16595804/c7ed84cb-76cb-41b3-b165-8f54351b7fc9)